internal/backends/sqlite/database.go

```diff
@@ -218,8 +230,21 @@ func relationStats(ctx context.Context, db *fsql.DB, list []*metadata.Collection
 		return nil, lazyerrors.Error(err)
 	}
 
-	// TODO https://github.com/FerretDB/FerretDB/issues/3293
-	stats.countIndexes, stats.sizeIndexes = 0, 0
+	stats.countIndexes = indexes
+
+	q = fmt.Sprintf(`
+		SELECT
+		    SUM(pgsize) AS SizeIndexes
+		FROM dbstat
+		WHERE name IN (%s) AND aggregate = TRUE`,
+		strings.Join(indexesPlaceholders, ", "),
+	)
+
+	if err = db.QueryRowContext(ctx, q, indexesArgs...).Scan(
+		&stats.sizeIndexes,
+	); err != nil {
```

Comment:
```suggestion
	if err = db.QueryRowContext(ctx, q, indexesArgs...).Scan(&stats.sizeIndexes); err != nil {
```

We don't write
```go
fmt.Println(
  1
)
```
aren't we?